frontend/src/Components/App.module.css

```diff
@@ -1,12 +1,29 @@
 :root {
   --border-radius: 0.5rem;
   --button-width: 90vw;
-  --tng-blue: rgb(5, 99, 165);
-  --light-blue: hsl(205, 95%, 40%);
-  --tng-gray: rgb(161, 166, 174);
-  --light-gray: hsl(217, 8%, 94%);
-  --dark-red: hsl(0, 100%, 40%);
-  --red: hsl(0, 100%, 50%);
+  --primary: rgb(5, 99, 165);
+  --primary-light: hsl(205, 95%, 40%);
+  --secondary: hsl(217, 8%, 94%);
+  --tertiary: rgb(161, 166, 174);
```

Comment:
Could we at least add comments where these colors come from? The TNG colors are specifically taken to be the exact colors of the TNG logo. This information is lost here, which I find problematic.

Reply:
Sure, I will add comments. The main reason why I removed the label, is that the TNG blue is a little bit hard to spot on the dark background. Therefore, I increased the brightness a bit on the dark color scheme.